nengo/tests/test_probe.py

```diff
@@ -219,6 +219,20 @@ def test_solver_defaults():
     assert f.solver is solver3
 
 
+def test_ensemble_encoders():
+    """Check encoders probed from ensemble are correct"""
+    with nengo.Network() as model:
+        ens = nengo.Ensemble(n_neurons=10, dimensions=2, radius=1.5)
+        p_enc = nengo.Probe(ens, 'scaled_encoders')
+
+    with nengo.Simulator(model) as sim:
```

Comment:
In the future, use the `Simulator` pytest fixture here rather than instantiating `nengo.Simulator` directly. You can see how this is done in other tests in this file, like `test_dts`. I'll fix this for now though :)